docs/verifiable-document/overview.md

```diff
@@ -4,8 +4,37 @@ title: Verifiable Document Overview
 sidebar_label: Overview
 ---
 
-Overview
+Verifiable documents forms the core of the OpenAttestation(OA) Framework. In this quick start guide, you will be deploying your first verifiable document.
 
-## Header
+## Goal
 
-TBD
+By the end of this tutorial you would be able to create your "Certificate of Completion" that is valid on any OA Viewer connected to the Ethereum ropsten network. From there, you may:
+
+1. Change the data structure of the document to suite your needs
+1. Change the rendering of the document to reflect your document styles
+1. Change the backend to use Ethereum main network
+
+## Overview of Components
+
+![Overview of Components](/docs/verifiable-document/overview/overview.png)
+
+### Document Store Smart Contract
+
+When a document is being issued, a proof of the issuance is store onto the Ethereum blockchain. A smart contract is used to provide a globally consistent record for anyone to query a given document's issuance status.
+
+### DNS Records
+
+A domain is required to issue an OA file. A DNS record will be inserted to the DNS to assert the identity of the OA document creator.
```

Comment:
I dunno if must be is better than will be in that case ?

will be make it sounds we will do it
must be makes it sounds it's mandatory

I leave it to you